publish-pr-preview/entrypoint.sh

```diff
@@ -73,21 +94,178 @@ const last_line = `Once the branch associated with this tag is deleted (usually
 
 markdown(`${first_line}\n${second_line}\n\`\`\`bash\n${install_tag}\n\`\`\`\n${fourth_line}\n\`\`\`bash\n${update_json}\n\`\`\`\n${last_line}`)
 EOT
+    else
+cat << "EOT" > dangerfile.js
+const { markdown } = require('danger');
+const pjson = require('./published.json');
 
-  fi
+function packages(){
+  let stringy = "";
+  pjson.packages.map(x=>{
+    stringy += package(x.name)
+  })
+  return stringy;
```

Comment:
You're using map like forEach. You have here a reduce operation. 

```js
let packages = pjson.packages.reduce((acc, item) => acc + package(item.name), '');
```

Reply:
I'll fix that now.